pyproject.toml

```diff
@@ -0,0 +1,67 @@
+[project]
+name = "Willow"
+description = "A Python image library that sits on top of Pillow, Wand and OpenCV"
+authors = [{name = "Karl Hobley", email = "karl@kaed.uk"}]
```

Comment:
Just wondering: should we add a [`maintainers`](https://peps.python.org/pep-0621/#authors-maintainers) key and list the Wagtail Core team as maintainers? Since Karl has left the Wagtail space.

Reply:
It is a good suggestion, but let's tweak this in a later PR as I don't have a good list :D

Reply:
I did not mean to list all members of the team since people come and go all the time. I meant a more generic `Wagtail Core team <hello@wagtail.org>` entry.

Reply:
ah! will address separately

Reply:
165078e